core/os/os2/file_util.odin

```diff
@@ -107,17 +107,17 @@ read_entire_file :: proc{
 }
 
 @(require_results)
-read_entire_file_from_path :: proc(name: string, allocator: runtime.Allocator) -> (data: []byte, err: Error) {
+read_entire_file_from_path :: proc(name: string, allocator: runtime.Allocator, loc := #caller_location) -> (data: []byte, err: Error) {
 	f, ferr := open(name)
 	if ferr != nil {
 		return nil, ferr
 	}
 	defer close(f)
-	return read_entire_file_from_file(f, allocator)
+	return read_entire_file_from_file(f=f, allocator=allocator, loc=loc)
 }
 
 @(require_results)
-read_entire_file_from_file :: proc(f: ^File, allocator: runtime.Allocator) -> (data: []byte, err: Error) {
+read_entire_file_from_file :: proc(f: ^File, allocator: runtime.Allocator, loc := #caller_location) -> (data: []byte, err: Error) {
```

Comment:
I think I need to add this on the `master` branch because it'll be useful there too.